dali/CMakeLists.txt

```diff
@@ -66,18 +66,11 @@ target_link_libraries(${dali_lib} PRIVATE "-Wl,--exclude-libs,${exclude_libs}")
 message(STATUS "Adding dependencies to ${dali_lib}: '${DALI_LIBS}'")
 target_link_libraries(${dali_lib} PRIVATE ${DALI_LIBS})
 
-# install libdali to build dir
-set_target_properties(${dali_lib} PROPERTIES
-    LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib
-)
-
-# filter all *.h which don't have 'test' in name, create pairs 'src dst' string with sed and call xargs
-# with install command to copy with dest recursive dir creation
-add_custom_command(
-    TARGET ${dali_lib} POST_BUILD
-    COMMAND find ${CMAKE_CURRENT_LIST_DIR} -regex ".*\\\\.h" | grep --invert test
-    | sed -r 's,\(^${CMAKE_CURRENT_LIST_DIR}\)\(.*\),\\1\\2 ${CMAKE_BINARY_DIR}/include/dali\\2,'
-    | xargs -n 2 install -D)
+# libdali installation paths
```

Comment:
The whole idea was to remove the installation step, so the user can just issue `make` . For install by default it goes to system dirs, for other destination cmake parameter need to be passed and there was lot of questions about that.
So please don't change this one.

Reply:
My fault. I didn't mean to change this file. I will do a rollback